src/code/z_player_lib.c

```diff
@@ -1,7 +1,100 @@
 #include <ultra64.h>
 #include <global.h>
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_player_lib/func_8008E750.s")
+typedef struct {
+    u8 unk_0;
+    u16 unk_2;
+} Struct_8008F2F8;
+
+// TODO decompile data
+
+extern SkeletonHeader* D_80125B70[];
+
+extern s16 D_80125B78[];
+
+extern u8 D_80125C44[];
+
+extern Struct_8008F2F8 D_80125C88[];
+
+extern u8 D_80125C98[];
+
+extern u32 D_80125F18[];
+
+extern u32 D_80125F20[];
+
+extern u32 D_80125F28[];
+
+extern u32 D_80125F30[];
+
+extern u32 D_80125F38[];
+
+extern u32* D_80125F40[];
+
+extern u8 D_8012607C[];
+
+extern f32 D_801260D0;
+
+extern Vec3f D_801260A4;
+extern Vec3f D_801260B0;
+extern Vec3f D_801260BC;
+extern Vec3f D_801260C8;
+
+extern Vec3f D_80126080;
+extern Vec3f D_8012608C;
+extern Vec3f D_80126098;
+
+// TODO decompile bss
+
+extern u8 D_80160008[]; // TODO check type
+
+// Segment Addresses
+
+extern LinkAnimetionEntry D_04003238;
+extern u32 D_0602A738;
+extern u32 D_0602CB48;
```

Comment:
if youre not sure what this is it should probably be UNK_TYPE. same goes for anything in the player instance struct